easybuild/easyconfigs/m/MITObim/MITObim-1.9.1-foss-2018b.eb

```
@@ -0,0 +1,33 @@
+easyblock = "CmdCp"
+
+name = 'MITObim'
+version = '1.9.1'
+
+homepage = "https://github.com/chrishah/MITObim"
+description = """The MITObim procedure (mitochondrial baiting and iterative mapping) represents
+ a highly efficient approach to assembling novel mitochondrial genomes of non-model organisms 
+ directly from total genomic DNA derived NGS reads."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://raw.githubusercontent.com/chrishah/MITObim/v1.9.1/']
+sources = [{'filename': 'MITObim.pl', 'extract_cmd': "cp %s ."}]
+checksums = ['8f83be5391db8a282e8c193b23e53c8d99c9894fb6990ea06de1321207ba6dd3']
+
+dependencies = [
+    ('Perl', '5.28.0'),
+    ('MIRA', '4.0.2'),
+]
+
+cmds_map = [('MITObim.pl', "chmod a+rx MITObim.pl")]
```

The permissions of this file in the tarball are already properly set. On the other hand, you can skip the build step altogether
```suggestion
skipsteps = ['build']
```